docs/architecture.md

```diff
@@ -0,0 +1,91 @@
+# Architecture
+
+The code review bot is not a single piece, but a distributed system using several providers.
+
+![](architecture.png)
```

Comment:
Where is the mermaid file for this one?

Reply:
It's made with draw.io . I added the source